libraries/ms-common/src/stm32f0xx/spi.c

```diff
@@ -0,0 +1,150 @@
+#pragma once
+
+#include "spi.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "gpio.h"
+#include "status.h"
+#include "stm32f0xx.h"
+#include "stm32f0xx_rcc.h"
```

Comment:
You don't need to include these. Including `stm32f0xx.h` includes all associated `stm32f0xx____.h` files as well.